### DISCUSSION TITLE
Squares when open a window to import colection

### DISCUSSION THREAD

**Discussion by wilsntn** [usebruno/bruno#2172] [Q&A] [answered]
Im using buntu 23 and when i try to import a collection my file explorer shows like this
![Screenshot from 2024-04-25 11-06-26](https://github.com/usebruno/bruno/assets/91102332/143f3178-529d-4b6b-ae4a-4fd5a5991424)
Is this related to my SO i guess but can someone help me?

**Comment by pietrygamat** [comment]
It's most likely to be solved in #2154, isn't it?

**Reply by wilsntn** [reply, marked as answer]
i had installed with flatpack.. i removed then installed with .deb from releases page and solved the problem

**Comment by parrotcar00** [comment]
I'm still seeing this issue on Ubuntu with a snap installation.
